.classpath

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
.classpath dosyasini .gitignore'a ekle be bu dosyayı PRdan sil. Bu tür dosyalar senin kendi development environmentın ile ilgili dosyalar ve bu tür dosyalar code repoda tutulmaz.